README.md

```diff
@@ -9,13 +9,59 @@ git submodule update
 # Install tcmalloc according to
 # https://github.com/gperftools/gperftools/blob/master/INSTALL
 
-pushd redis && env USE_TCMALLOC=yes make -j && popd
-pushd glog && cmake . && make -j install && popd
-pushd leveldb && make -j && popd
+cd redis
+env USE_TCMALLOC=yes make -j
+cd ..
+
+cd glog 
+cmake . 
+make -j install 
+cd ..
+
+cd leveldb 
+make -j 
+cd ..
+
+# Install grpc. A suitable version is included as a submodule of etcd3 but you
+# can install the latest version separately on the system too.
+cd etcd3
+git submodule update --init
+cd grpc 
+git submodule update --init 
+make 
+make install
+cd ../..
+```
```

Comment:
After grpc is installed I try to build credis.  It then goes into `credis/etcd3/CMakeLists.txt`, which contains `10:find_package(Protobuf REQUIRED)`.  This then failed on my Mac because cmake can't find it installed.  

Let's add protobuf installation instructions to this README, or try to include protobuf as a git submodule?  Up to you.

Reply:
BTW, L9 comments that FindProtobuf can be used, but L10 uses `find_package` :)